CHANGES.md

```diff
@@ -83,6 +83,7 @@ be compatible with this version.
     when messages are received.  [[#3273]]
 
 ### Bug fixes
+ -  Fixed a broken backwards compatibility of Currency  [[#3284]]
```

Comment:
```suggestion
 -  Fixed a broken backwards compatibility of Currency  [[#3268], [#3284]]
```

I couldn't suggest on not edited lines. You should manually add the below line.

```
[#3268]: https://github.com/planetarium/libplanet/issues/3268
```

<img width="625" alt="image" src="https://github.com/planetarium/libplanet/assets/26626194/8a5d6f1e-4930-4b42-ad0f-8279fc732271">